### PR TITLE
[loopback-2.x] Fix CI

### PR DESCRIPTION
**Pull request by dhmlau**
### Description
Fix CI for1.x branch in order for juggler 2.x CI to pass.

- Drop support of node 0.10 and 0.12
    - backport #23 

#### Related issues
connect to: strongloop/loopback-datasource-juggler#1342

### CODE REVIEW

**Comment by dhmlau** [comment]
@slnode  test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by ssh24** [comment]
@dhmlau Rebased the branch for you. Not sure what the status of this PR is. Are we going to proceed with this?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+ - 4
+ - 6
```
Comment (ssh24) [inline .travis.yml]: Need a new line here.

**Comment by dhmlau** [comment]
@ssh24, based on what we've found here (https://github.com/nodejs/LTS#readme), it should be good to proceed.  Let's get the confirmation from @raymondfeng before landing.

**Comment by dhmlau** [comment]
@slnode test please

**Pull request merged**